### ISSUE TITLE
Forum v2 feature request: move post

### ISSUE THREAD

**Issue by mochet** [Joystream/joystream#3000]
Based on what is described here: https://joystream.gitbook.io/joystream-handbook/subsystems/forum

There is no functionality to move a post from one thread to another. This is a good (I would argue almost essential) feature as it allows moderators to ensure threads remain on-topic, or if users post in the wrong area (perhaps just due to a lack of knowledge) the post can be moved to another thread.

In summary: Would be nice to have, but not critical

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201703380142370/1201703600467603) by [Unito](https://www.unito.io)
